src/manifest.json

```diff
@@ -52,7 +52,8 @@
   "permissions": [
     "activeTab",
     "storage",
-    "menus"
+    "menus",
+    "clipboardRead"
```

Comment:
Can you make it an optional permission (see `optional_permissions`), so we don't request that permission from anyone?

See what I wrote before:
> If a permission is required, one can couple it in this extension [with the PermissionRequest](https://github.com/TinyWebEx/PermissionRequest) library (also used in other extensions, if one needs to get an example).

https://github.com/rugk/offline-qr-code/issues/282#issuecomment-1199975970

As a comparison, see [how the awesome emoji picker implements this](https://github.com/rugk/awesome-emoji-picker/blob/26295d9b391ee6a48b2e636f1339e109989f5f44/src/options/modules/CustomOptionTriggers.js#L317-L334) for clipboard pasting in that case:

![image](https://user-images.githubusercontent.com/11966684/196689664-81819870-8719-4f6d-8963-e896851fad1f.png)